src/Pages/Profile/MemberView/Profile.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react';
+import './ProfileModifier.css';
+import axios from 'axios'
+import Footer from '../../../Components/Footer/Footer.js'
+import InfoCard from './InfoCard.js'
+
+export default class Profile extends Component {
+
+  constructor(props) {
+    super(props)
+
+    // Variables that will be send to data base
+    this.state = {
+      user: '',
+      fullName: ''
+    }
+  }
+
+  componentDidMount() {
+    axios
+      // get user
+      .post('/api/user/search', {
+        // don't need email
+        token: this.props.user.token,
+        email: this.props.user.email
+      })
+      .then(result => {
+        if (result.status >= 200 && result.status < 300) {
+
+          //concat user to full name
+          this.setState({ user: result.data }, () => {
+            const name = this.state.user.firstName[0].toUpperCase() +
+              this.state.user.firstName.slice(1, this.state.user.firstName.length) +
+              " " +
+              this.state.user.lastName[0].toUpperCase() +
+              this.state.user.lastName.slice(1, this.state.user.lastName.length)
+
+            this.setState({ fullName: name })
+          })
+
+        }
+      })
+      .catch(err => {
+        console.log(err)
+      })
+  }
+
+  render() {
+    const fields =
+      (this.state.user) ?
+        [
+          { front: 'Name', end: this.state.fullName },
```

Comment:
Let's rename the fields here to something more descriptive, e.g.
```suggestion
          { title: 'Name', value: this.state.fullName },
```